extensions/src/main/scala/pl/touk/nussknacker/extensions/db/schema/TableDefinition.scala

```diff
@@ -0,0 +1,18 @@
+package pl.touk.nussknacker.extensions.db.schema
+
+import pl.touk.nussknacker.engine.api.typed.typing.{TypedClass, TypedObjectTypingResult, TypingResult}
+
+import java.sql.ResultSetMetaData
+
+object TableDefinition {
+
+  def apply(resultMeta: ResultSetMetaData): TableDefinition =
+    TableDefinition(
```

Comment:
Can we do it in one line? :)

Reply:
We can,.. but does it make a difference?

Reply:
done